src/applications/vaos/new-appointment/redux/helpers/formSubmitTransformers.js

```diff
@@ -1,238 +1,167 @@
-
+  const facilityTimezone = getTimezoneByFacilityId(data.communityCareSystemId);
   return {
-    typeOfCare: typeOfCare.ccId,
-    typeOfCareId: typeOfCare.ccId,
-    appointmentType: typeOfCare.name,
-    facility: {
-      name: organization.name,
-      facilityCode: organization.id,
-      parentSiteCode: organization.vistaId,
+    kind: 'cc',
+    status: 'proposed',
+    locationId: data.communityCareSystemId,
+    serviceType: typeOfCare.idV2 || typeOfCare.ccId,
+    // comment: data.reasonAdditionalInfo,
+    reasonCode: getReasonCode({
+      data,
+      isCC: true,
+      isDS: false,
+    }),
+    contact: {
+      telecom: [
+        {
+          type: 'phone',
+          value: data.phoneNumber,
+        },
+        {
+          type: 'email',
+          value: data.email,
+        },
+      ],
     },
-    purposeOfVisit: CC_PURPOSE,
-    phoneNumber: data.phoneNumber,
-    verifyPhoneNumber: data.phoneNumber,
-    // The bad camel casing here is intentional, to match downstream
-    // system
-    bestTimetoCall: Object.entries(data.bestTimeToCall)
+    requestedPeriods: data.selectedDates.map(date => ({
+      start: moment
+        .tz(date, facilityTimezone)
+        .utc()
+        .format(),
+      end: moment
+        .tz(date, facilityTimezone)
+        .utc()
+        .add(12, 'hours')
+        .subtract(1, 'minute')
+        .format(),
+    })),
+    // These four fields aren't in the current schema, but probably should be
+    preferredTimesForPhoneCall: Object.entries(data.bestTimeToCall || {})
       .filter(item => item[1])
       .map(item => titleCase(item[0])),
-    preferredProviders,
-    newMessage: data.reasonAdditionalInfo,
     preferredLanguage: LANGUAGES.find(
       lang => lang.id === data.preferredLanguage,
     )?.value,
-    ...getRequestedDates(data),
-    ...cityState,
-    // defaulted values
-    visitType: 'Office Visit',
-    requestedPhoneCall: false,
-    email: data.email,
-    officeHours: [],
-    reasonForVisit: '',
-    distanceWillingToTravel: 40,
-    secondRequest: false,
-    secondRequestSubmitted: false,
-    inpatient: false,
-    status: 'Submitted',
-    providerId: '0',
-    providerOption: '',
+    preferredLocation,
+    practitioners,
   };
 }
 
-export function transformFormToAppointment(state) {
+export function transformFormToVAOSVARequest(state) {
   const data = getFormData(state);
-  const clinic = getChosenClinicInfo(state);
-  const timezone = getTimezoneByFacilityId(data.vaFacility);
+  const typeOfCare = getTypeOfCare(data);
 
+  return {
+    kind: data.visitType,
+    status: 'proposed',
+    locationId: data.vaFacility,
+    // This may need to change when we get the new service type ids
+    serviceType: typeOfCare.idV2,
+    reasonCode: getReasonCode({
+      data,
+      isCC: false,
+      isDS: false,
+    }),
+    // comment: data.reasonAdditionalInfo,
+    requestedPeriods: [
+      {
+        start: moment.utc(data.selectedDates[0]).format(),
```

Comment:
since we want to eventually get rid of moment js, can we refactor this now?

Reply:
Hmm I haven't looked into that but would it be better to work on the momentjs => date-fns refactor in its entirety in a separate PR instead of spreading it over several unrelated PRs?

Reply:
Generally I don't like to combine PRs (unless they are sufficiently small) just in case we ever need to revert a commit. I think this PR, though simple, is big enough I would prefer not including additional concerns. What do you think?

Reply:
@JunTaoLuo - I don't mind that approach